classes/Project/Project.mjs

```diff
@@ -45,7 +50,39 @@ export default class Project {
     return database.remove(projectId)
   }
 
-  checkUserAccess(userAgent) {
+  async addMember(email, rolesString) {
+    try {
+      let user = await User.getByEmail(email)
+      const roles = this.parseRoles(rolesString)
+      let updatedProject
+      let message = `You have been invited to the TPEN project ${
+        this.projectData?.name
+      } with the following role(s): ${roles.join(", ")}.`
```

Comment:
Not important for this PR, but I think we don't need the roles in the original "Join this project" message. I can separately think about some scenarios and suggest copy.